docs/app.md

```diff
@@ -190,6 +191,31 @@ myApp.isRunning();
 myApp.start();
 ```
 
+### App `restart`
+
+This method sets the `App` to a not running state, then sets it to a running state.
```

Comment:
Super nitpicky, but how about:

This method saves the current state of the app before stopping it. It then starts the app again with the preserved state attributes.